packages/clarity-js/src/performance/observer.ts

```diff
@@ -81,6 +83,10 @@ function process(entries: PerformanceEntryList): void {
                 break;
             case Constant.LCP:
                 if (visible) { metric.max(Metric.LargestPaint, entry.startTime); }
```

Comment:
do we want to only log targetNode  if it is visible? not sure why we log the metric when visible, but the rest of it regardless